ISISComputingGroup/ibex_gui
# Ticket1756: Connect to NICOS in the GUI

### Description of work

This work has refactored the activeMQ connections in the GUI so that they can be used by NICOS as well as the log messages. NICOS is now connected to when the perspective is opened. See https://github.com/ISISComputingGroup/IBEX/issues/1756 

### To test

* Ensure that receiving real-time log messages still works on your instrument and when pointing to others.
* When clicking on the NICOS scripting perspective NICOS login data is printed to the console
* Login data is again printed when switching between instruments with the NICOS window open.

NOTE: for the login data to be received you will need to have your admin creds in conf/activemq.xml set to "username", "password" or change the credentials hardcoded into MQConnection to those in activemq.xml

---

#### Code Review


### Functional Tests


### Final Steps
